client-core/src/wallet/syncer.rs

```diff
@@ -750,6 +860,56 @@ fn filter_incomming_staking_transactions<'a>(
     Ok(Default::default())
 }
 
+fn make_light_client_instance(
+    peer_id: PeerId,
+    addr: tendermint::net::Address,
+    db_path: impl AsRef<Path>,
+) -> Instance {
+    let mut peer_map = HashMap::new();
+    peer_map.insert(peer_id, addr);
+
+    let timeout = Duration::from_secs(10);
+    let io = ProdIo::new(peer_map, Some(timeout));
+
+    let db = sled::open(db_path).unwrap_or_else(|e| {
+        println!("[ error ] could not open database: {}", e);
+        std::process::exit(1);
+    });
+
+    let mut light_store = SledStore::new(db);
```

Comment:
so the light client has a separate storage? any guarantees that this won't conflict with client's internal storage (e.g. have some root path and always post-fix it?)?

Reply:
There's separate db for each peer, the db path is `{wallet_db_path}/{peer_id}/`
https://github.com/crypto-com/chain/pull/1901/files#diff-873b16429c2cd004665e7ef924cf4643R1272